static/docs/user-guide/dvcignore.md

```diff
@@ -7,32 +7,26 @@ Sometimes you might want DVC to ignore files while traversing the project
 directory. For example, when working on a project with many files in its data
 directory, you might encounter extended execution time for operations that are
 as simple as `dvc status`. To prevent this, we are implementing `.dvcignore`
-files handling. When fully implemented, their implementation is intended to
-provide similar functionality as `.gitignore` files provide for `git`.
+files handling. `.dvcignore` by design works similar way as `.gitignore` does.
 
 ## How does it work
 
 - You need to create `.dvcignore` file.
 - Populate it with [patterns](https://git-scm.com/docs/gitignore) that you would
-  like to ignore.
+  like to ignore
 - Each line should contain only one pattern.
 - During execution of commands that traverse directories, DVC will ignore
   matching paths.
-- Not every operation supports `.dvcignore`. To see current limitations, read
-  following paragraph.
-
-## Current limitations
-
-During development, we noticed that there are few potential uses cases that
-might be tricky to handle (e.g. what to do when we are `dvc add`-ing directory
-containing `.dvcignore` file). Therefore, we decided to enable this feature
-gradually in different parts of the project.
-
-Currently `.dvcignore` files will be read and applied in any operation that
-collects DVC-files (e.g. `checkout`, `metrics`, `status`, `run`, `repro`), so it
-is advised to use it in cases described in the first paragraph, when amount of
-files in tree of repository directory causes performance issues.
-
+- Ignored files will not be cached, they will be non-existent for DVC. It's
+  worth to remember that, especially when ignoring files inside DVC-handled
+  directories.
+- Keep in mind, that when you add to dvcignore entries that affect one of the
+  existing outputs, its status will change and DVC will behave as if that
```

Comment:
again, better to use terms like `existing files or directories (outputs) that are controlled by DVC, corresponding DVC-files status ...`

Or alternatively you can use <abbr>outputs</abbr> and add new term to the glossary.js file. It's quite easy :)

Reply:
you need to put `<abbr>` around outputs to make it work